app/helpers/foreman_host_reports/reports_dashboard_helper.rb

```diff
@@ -0,0 +1,88 @@
+module ForemanHostReports
+  module ReportsDashboardHelper
+    def host_reports_searchable_links(name, search, counter, format)
+      content_tag :li do
+        content_tag(:span, raw('&nbsp;'), :class => 'label', :style => "background-color:" + host_reports_report_color[counter]) +
+          raw('&nbsp;') +
+          link_to(name, hosts_path(:search => search), :class => "dashboard-links") +
+          content_tag(:h4, totals(format)[counter])
```

Comment:
@lzap Because of the use of totals here, I kept the function but I changed the usage in the html.erb files. 
Not sure what did you suggested regarding the format argument, it is required here and in the other helper functions.

Reply:
I see, well this ERB is correct, however, I will drop a comment below.